libs/damap/src/assets/i18n/templates/en.json

```diff
@@ -5,10 +5,12 @@
         "input": {
           "title": "Project title",
           "description": "Project description",
-          "button": "Update project",
-          "acronym": "Update acronym",
+          "button": "Project name",
```

Comment:
This key affects the button and the label of the name input field - only the label should be chnaged to project name though